src/routes/profile.svelte

```diff
@@ -1,15 +1,122 @@
+<script context="module">
+  import getInitialData from '@/utils/get-initial-data.js';
+  import isoForURL from '@/utils/iso-for-url.js';
+
+  export async function preload(page, session) {
+    let timelineFetchedUntil = new Date();
+    timelineFetchedUntil.setMonth(timelineFetchedUntil.getMonth() - 3);
+    const {
+      account,
+      timeline,
+      statistics,
+      notificationSettings,
+      competences,
+    } = await getInitialData(this, session, new Map([
+      ['account', '/account'],
+      ['timeline', `/account/timeline?start_date=${isoForURL(timelineFetchedUntil)}`],
+      ['statistics', `/account/statistics?start_date=${isoForURL(timelineFetchedUntil)}`],
+      ['notificationSettings', '/account/notification_settings'],
+      ['competences', '/competences'],
+    ]));
+    if (account == null) {
+      this.error(403, 'Profile');
+    }
+    return {
+      account,
+      timeline,
+      statistics,
+      notificationSettings,
+      timelineFetchedUntil,
+      competences,
+    };
+  }
+</script>
+
 <script>
-  import Layout from '@/layouts/auth.svelte';
+  import { stores } from '@sapper/app';
+  import Layout from '@/layouts/default.svelte';
   import Info from '@/containers/profile/info.svelte';
-  import Tabs from '@/containers/profile/tabs.svelte';
+  import Tabs from 'ui/tabs.svelte';
   import Timeline from '@/containers/profile/timeline.svelte';
-  import Statistics from '@/containers/profile/stats.svelte';
+  import Statistics from '@/containers/profile/statistics.svelte';
   import Notifications from '@/containers/profile/notifications.svelte';
+  import * as api from '@/utils/api.js';
+  import tabs from '@/constants/profile/tabs.js';
+
+  const { session } = stores();
 
-  let activeTab = 'T';
-  function changeTab(target) {
+  export let account;
+  export let timeline;
+  export let statistics;
+  export let notificationSettings;
+  export let timelineFetchedUntil;
+  export let competences;
+  $session.user = account;
+
+  let timelinePromises = (timeline.data.length ? [new Promise(resolve => resolve(timeline))] : []);
```

Comment:
Try to improve readability:
```suggestion
  let timelinePromises = [];
  if (timeline.data.length)
    timelinePromises.push(new Promise(resolve => resolve(timeline)));
```